config/default.yaml

```diff
@@ -95,8 +95,8 @@ executor:
             # The host or IP of the kubernetes cluster
             host: kubernetes.default
             # The jwt token used for authenticating kubernetes requests
-            # Loaded from /var/run/secrets/kubernetes.io/serviceaccount/token by default
-            # token:
+            # Loaded from /var/run/secrets/kubernetes.io/serviceaccount/token by default if not specified
+            token: 'fakeToken'
```

Comment:
@FenrirUnbound That's a good point! Changed the k8s executor to not complain if no token passed in and path not exists.
https://github.com/screwdriver-cd/executor-k8s/pull/61